apps/src/templates/instructions/RubricField.jsx

```diff
@@ -65,16 +74,14 @@ class RubricField extends Component {
     return (
       <div style={performanceHeaderStyle}>
         {this.props.showFeedbackInputAreas && (
-          <input
-            type={'checkbox'}
+          <CheckRadioButton
             //Concatenate because injecting the string seems to result in all of
-            //the input elements having the name id
+            //the input elements having the same id
             id={'rubric-input-' + this.props.rubricLevel}
-            name="rubric"
             value={this.props.rubricLevel}
             checked={this.props.currentlyChecked}
-            onChange={this.handleRubricChange}
-            disabled={this.props.disabledMode}
+            onRadioButtonChange={this.handleRubricChange}
```

Comment:
you should be able to simplify this line to:

```suggestion
            onRadioButtonChange={this.props.onChange}
```

which means you can remove the `handleRubricChange` method and the binding on line 63. does this not work? (for background, i was reading about this same topic [here](https://medium.com/@nikolalsvk/loosing-bind-this-in-react-8637ebf372cf))